website/docs/r/network_routed_v2.html.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_network_routed_v2"
+sidebar_current: "docs-vcd-resource-network-routed-v2"
+description: |-
+  Provides a VMware Cloud Director Org VDC routed Network. This can be used to create, modify, and
+  delete internal networks for vApps to connect backed by NSX-T and NSX-V.
+---
+
+# vcd\_network\_routed\_v2
+
+Provides a VMware Cloud Director Org VDC routed Network. This can be used to create, modify, and
+delete internal networks for vApps to connect backed by NSX-T and NSX-V.
+
+Supported in provider *v3.2+* for both NSX-T and NSX-V VDCs.
+
+## Example Usage (NSX-T backed routed Org VDC network)
+
+```hcl
+resource "vcd_network_routed_v2" "nsxt-backed" {
+  org  = "my-org"
+  vdc  = "my-nsxt-org-vdc"
+  name = "nsxt-routed 1"
+  description = "My first routed Org VDC network backed by NSX-T"
```

Comment:
```suggestion
  description = "My routed Org VDC network backed by NSX-T"
```
It was nice, but I think will get old soon :)

Reply:
Changed.